lib/movie_presentation.dart

```diff
@@ -0,0 +1,72 @@
+import 'package:flutter/material.dart';
+import 'star_rate.dart';
+import 'styles.dart';
+
+const Color starColor = Colors.yellow;
+
+class MoviePresentation extends StatelessWidget {
+  const MoviePresentation({
+    super.key,
+    required this.movieBackDrop,
+    required this.movieOriginalTitle,
+    required this.movieRate,
+  });
+
+  final String movieBackDrop;
+  final String movieOriginalTitle;
+  final double movieRate;
+
+  static const double borderRadiusBackDrop = 20;
+  static const double imageHeightBackDrop = 220;
+  static const double leftPaddingListView = 15;
+  static const double leftPaddingImageOriginalTitle = 30;
+  static const double sizeofOriginalTitleBox = 200;
+
+  static const int originalTitleMaxLines = 3;
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.only(left: leftPaddingListView),
+      child: ListView(
+        physics: const NeverScrollableScrollPhysics(),
+        shrinkWrap: true,
```

Comment:
Same as before, `ListView` not needed. Use `Column` instead

Reply:
Done!